aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -210,6 +215,24 @@ public boolean isMetricsEnabled() {
     /** Override value for System.currentTimeInMillis.  Used for unit testing reconnect logic. */
     private Long unitTestMillisOverride;
 
+    /**
+     * Sets the usernameFields map.
+     *
+     * @param usernameFields usernameFields map
+     */
+    public void setUsernameFields(Map<String, String> usernameFields) {
+        this.usernameFields = usernameFields;
+    }
+
+    /**
+     * Returns the usernameFields map
```

Comment:
Why do we need a getter for this field?

Reply:
I can't think of a use-case for this. Removed.